reframe/core/schedulers/slurm.py

```diff
@@ -12,6 +12,7 @@
 from argparse import ArgumentParser
 from contextlib import suppress
 
+import reframe.core.logging as logging
```

Comment:
I think this is unused.

Reply:
It's used for the logging of the error. If I use `self.log` the logged message will not appear on the output. For this reason I log it as `logging.WARNING`, because it's useful to be able to see that the system was not stable.